packages/team-management/src/components/TeamsAddEdit/index.tsx

```diff
@@ -90,6 +91,73 @@ export const defaultProps = {
   opensrpBaseURL: '',
 };
 
+/**
+ * function to fetch a paginated practitioner object
+ *
+ * @param opensrpBaseURL - OpenSRP API base URL
+ * @param practitionersEndpoint - OpenSRP practitioners endpoint
+ * @param pageNumber - paginated page number
+ * @param pageSize - paginated page size
+ * @param errorNotification - the error message to show for a failed request
+ * @returns {Promise<Practitioner[]>} an array of practitioners
+ */
+async function fetchPractitioners(
+  opensrpBaseURL: string,
+  practitionersEndpoint: string,
+  pageNumber: number,
+  pageSize: number,
+  errorNotification: string = lang.ERROR_OCCURRED
+): Promise<Practitioner[]> {
+  const composePaginatedEndpoint = `${practitionersEndpoint}?pageNumber=${pageNumber}&pageSize=${pageSize}`;
```

Comment:
`OpensrpService.list` takes a param parameter that helps create the search params from an object.

Reply:
@p-netm how would you create a search param object from above?

Reply:
e.g elsewhere: https://github.com/opensrp/web/blob/416d657de70f62f8ef1e6583a7e1e728310895c8/packages/opensrp-plans/src/helpers/dataLoaders.ts#L453

Reply:
aff169d3df31d951cd73715adc4604ec0f6bfbe2